flexmeasures/ui/static/openapi-specs.json

```
@@ -7,7 +7,7 @@
     },
     "termsOfService": null,
     "title": "FlexMeasures",
-    "version": "0.31.0"
+    "version": "0.28.1"
```

```suggestion
    "version": "0.31.0"
```

Would happen anyway but good to fix now.

---

@joshuaunity This kept happening to me, too, while developing, due to our new precommit hook. A `pip show flexmeasures` probably shows you have an older version of FlexMeasures pip installed. I believe uninstalling it and doing an editable install instead (`pip install -e .`) should stop this from happening.